frontend/src/components/resourceMap/sources/definitions/workloadSource.tsx

```diff
@@ -0,0 +1,233 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Can you please add justifications next to any places where you disable eslint/tsc?

Reply:
removed

Reply:
thanks